### ISSUE TITLE
Crash during APK Compilation with ViewBinding Enabled in Sketchware

### ISSUE THREAD

**Issue by redowan178** [Sketchware-Pro/Sketchware-Pro#1515]
### Description:  
When using Sketchware to create an APK on my device (Poco C3, Android 10), the compilation process fails when ViewBinding is enabled. The issue occurs during the "compiling resources with aapt2" step and throws the following error:

---

**Error Log:**  
```
java.util.NoSuchElementException: List is empty.
	at kotlin.collections.CollectionsKt___CollectionsKt.first(_Collections.kt:221)
	at mod.pranav.viewbinding.ViewBindingBuilder.generateBindingForLayout(ViewBindingBuilder.kt:20)
	at mod.pranav.viewbinding.ViewBindingBuilder.generateBindings(ViewBindingBuilder.kt:13)
	at a.a.a.ProjectBuilder.generateViewBinding(ProjectBuilder.java:176)
	at com.besome.sketch.design.DesignActivity$BuildTask.doInBackground(DesignActivity.java:1099)
	at com.besome.sketch.design.DesignActivity$BuildTask.$r8$lambda$0AmVsWQJ9oulgeP8eMAUbl0tGkA(Unknown Source:0)
	at com.besome.sketch.design.DesignActivity$BuildTask$$ExternalSyntheticLambda3.run(D8$$SyntheticClass:0)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
	at java.lang.Thread.run(Thread.java:919)
```

---

### Steps to Reproduce:  
1. Open Sketchware on Poco C3 (Android 10).  
2. Enable ViewBinding for the project.  
3. Start the APK build process.  
4. Observe the error during the "compiling resources with aapt2" step.

---

### Expected Behavior:  
The APK should compile successfully with ViewBinding enabled.

---

### Actual Behavior:  
The compilation fails with the above error, specifically mentioning `java.util.NoSuchElementException: List is empty`.

---

### Additional Information:  
- **Device:** Poco C3  
- **Android Version:** 10  
- **Sketchware Version:** v7.0.0-SNAPSHOT-b6cf6cf-minApi26
The error only occurs when ViewBinding is enabled. Without ViewBinding, the APK compiles successfully.

**Comment by PranavPurwar** [comment]
this error has already been fixed way back

**Comment by redowan178** [comment]
Thank you for letting me know that this issue has been fixed previously. However, I am still encountering this problem with the following setup:  

- **Device:** Poco C3  
- **Android Version:** 10  
- **Sketchware Version:** "v7.0.0-SNAPSHOT-b6cf6cf-minApi26",

The error occurs specifically when ViewBinding is enabled, and I receive the following stack trace:  

```
java.util.NoSuchElementException: List is empty.
	at kotlin.collections.CollectionsKt___CollectionsKt.first(_Collections.kt:221)
	at mod.pranav.viewbinding.ViewBindingBuilder.generateBindingForLayout(ViewBindingBuilder.kt:20)
	...
```  

Could you please confirm:  
1. Which version contains the fix for this issue?  
2. If there are specific steps to ensure the fix is applied (e.g., updating to a newer version or applying patches)?  

I would appreciate your guidance to resolve this. Thank you!

**Comment by redowan178** [comment]
I'm still facing the error

**Comment by RoyTheReal** [comment]
Same

**Comment by PranavPurwar** [comment]
Fix available in #1539

**Comment by betermaher** [comment]
java.io.IOException: Cannot run program "/data/user/0/pro.sketchware/cache/aapt2": error=2, No such file or directory
	at java.lang.ProcessBuilder.start(ProcessBuilder.java:1050)
	at pro.sketchware.utility.BinaryExecutor.execute(BinaryExecutor.java:19)
	at mod.jbk.build.compiler.resource.ResourceCompiler$Aapt2Compiler.compileImportedResources(ResourceCompiler.java:396)
	at mod.jbk.build.compiler.resource.ResourceCompiler$Aapt2Compiler.compile(ResourceCompiler.java:132)
	at mod.jbk.build.compiler.resource.ResourceCompiler.compile(ResourceCompiler.java:62)
	at a.a.a.ProjectBuilder.compileResources(ProjectBuilder.java:185)
	at com.besome.sketch.design.DesignActivity$BuildTask.doInBackground(DesignActivity.java:1125)
	at com.besome.sketch.design.DesignActivity$BuildTask.$r8$lambda$0AmVsWQJ9oulgeP8eMAUbl0tGkA(Unknown Source:0)
	at com.besome.sketch.design.DesignActivity$BuildTask$$ExternalSyntheticLambda1.run(D8$$SyntheticClass:0)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:644)
	at java.lang.Thread.run(Thread.java:1012)
Caused by: java.io.IOException: error=2, No such file or directory
	at java.lang.UNIXProcess.forkAndExec(Native Method)
	at java.lang.UNIXProcess.<init>(UNIXProcess.java:133)
	at java.lang.ProcessImpl.start(ProcessImpl.java:141)
	at java.lang.ProcessBuilder.start(ProcessBuilder.java:1029)
	... 11 more

I am facing this problem in Sketchware Pro.
